tests/utils/dockercli/volumelifecycle.go

```diff
@@ -68,6 +68,24 @@ func AttachVolumeWithRestart(ip, volName, containerName string) (string, error)
 		dockercli.TestContainer)
 }
 
+// AttachWriteVolumeWithRestart Attach a volume with restart flag. Also create a file with specified name on volume
+func AttachWriteVolumeWithRestart(ip, volName, containerName, fileName string) (string, error) {
+	log.Printf("Attaching volume [%s] on VM[%s]\n", volName, ip)
+	writeCmd := " /bin/sh -c 'touch /vol1/" + fileName + "; sync ; tail -f /dev/null'"
+
+	return ssh.InvokeCommand(ip, dockercli.RunContainer+" --restart=always -d -v "+volName+
+		":/vol1 --name "+containerName+dockercli.ContainerImage+writeCmd)
+}
+
+// ListFilesOnVolume return output of list of files on volume
```

Comment:
same as above need to elaborate more... when to use? and assumption if you have made for this util.